Task2/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link href="css/content.css" rel="stylesheet" type="text/css">
+    <meta charset="UTF-8">
+    <title>Task2</title>
+</head>
+<body>
+<h1 id="Chap">Chapters</h1>
```

Comment:
Usually values of  `id`, `class`, `attribute` should be started with lower case.

Reply:
done